cmd/start.go

```diff
@@ -140,12 +141,21 @@ func Start(c *cli.Context) error {
 		operatorPrivateKey, operatorPublicKey,
 	)
 
+	keepManager, err := ethereumChain.BondedECDSAKeepManager()
+	if err != nil {
+		panic(fmt.Sprintf("Failed to resolve Keep manager: [%v].", err))
+	}
+
 	networkProvider, err := libp2p.Connect(
 		ctx,
 		config.LibP2P,
 		networkPrivateKey,
 		libp2p.ProtocolECDSA,
-		firewall.NewStakeOrActiveKeepPolicy(ethereumChain, stakeMonitor),
+		firewall.NewStakeOrActiveKeepPolicy(
+			ethereumChain.PublicKeyToOperatorID,
```

Comment:
This will be a fun spot for two or more p2p networks. Operator ID may be the same for Celo and Ethereum but we do not want Celo and Ethereum p2p client networks to mix with each other. We may, for example, expect that the string representation of Operator ID is prefixed with a network name... What do you think?

Reply:
Yep, the [note on KeepID](https://github.com/keep-network/keep-ecdsa/blob/faa12a770f65b7fecd48ff66ce35a69d8ed21be6/pkg/chain/chain.go#L40-L46) will apply across ids. I expect Ethereum ones will remain unprefixed for historical reasons, unless/until we can confirm that we can prefix safely. In particular, I want to make sure the refactor is network-compatible with the previous version.